src/app/common/stores/applications/Faucet/Wallet.store.ts

```diff
@@ -70,22 +72,71 @@ class WalletStore extends BaseStore implements Wallet {
    * @url https://docs.blocknative.com/onboard#initialization
    */
   initWalletHooks() {
-    this.faucetStore;
-    if (this.onboardSdk) return;
-    const connectionConfig = {
-      dappId: config.ONBOARD.API_KEY,
-      networkId: 5,
-      walletSelect: {
-        wallets,
+    const injected = injectedModule();
```

Comment:
We have a large code duplication of this code. (Faucet/Wallet.store.ts, Distribution/Wallet.store.ts, SsvWeb/Wallet.store.ts). Please refactor.

Reply:
will add ticket for refactoring this, because I have to add wallet connect, and after we can refactor it

Reply:
also the look the same in each store but still have some difference

Reply:
Please create ticket and paste here and resolve this conversation.

Reply:
nm, saw that you wrote in discord that you opened.